packages/core/upload/admin/src/components/AssetCard/AudioPreview.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable jsx-a11y/media-has-caption */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Box } from '@strapi/design-system/Box';
+import { VisuallyHidden } from '@strapi/design-system/VisuallyHidden';
+
+export const AudioPreview = ({ url, mime, alt }) => {
+  return (
+    <Box>
+      <audio controls>
+        <source src={url} type={mime} />
+      </audio>
+      <VisuallyHidden as="figcaption">{alt}</VisuallyHidden>
```

Comment:
Could this be simplified to:

```suggestion
      <audio controls src={src}>
        {alt}
      </audio>
```

? Then you'd not have to pass `mime` as a prop. Also the `figcaption` doesn't really fit here. Instead you could define the fallback text in the `<audio />` element directly ([source on MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/audio))

Reply:
Indeed more correct - ideally the same change would be applied to AssetPreview I guess but not sure how you stand vs disabling the linting rule.